util/pytest_log_filter.py

```diff
@@ -0,0 +1,16 @@
+import re
+import sys
+
+
+def filter_test_log(log):
+    with open(log) as f:
+        log_txt = f.read()
+    test_results = re.search("\n\n===(.+?)((.|\n)*)==", log_txt).group()  # Filter to only get the usefull parts of the info
+    test_results = test_results[test_results.find("="):]  # Remove preceding enters
+    # Only display when there is more to tell than just passed test cases (meaning failures, warning etc.)
+    if test_results.count("\n") > 0:
```

Comment:
`.count()` is going to scan the entire string, which can be slow if the string is long. You could write `"\n" in test_results`, which will automatically stop the scan as soon as one `\n` is found.